DAQ/CMakeLists.txt

```diff
@@ -129,6 +129,14 @@ cet_build_plugin(STMWaveformDigisFromFragments art::module
       BTrk_difAlgebra
 )
 
+cet_build_plugin(StrawRecoFromArtdaqFragments art::module
```

Comment:
@pavel1murat I think you made a typo here. It should be `StrawDigisFromArtdaqFragments`

Reply:
right !- thanks for catching that

Reply:
just fixed - thought that due to the triviality of the matter it has been already done